src/index.js

```diff
@@ -3,9 +3,20 @@ import ReactDOM from 'react-dom';
 import App from './App';
 import {unregister} from './registerServiceWorker';
 import {HashRouter} from 'react-router-dom';
+import createHistory from 'history/createBrowserHistory';
+
+import ReactGA from 'react-ga';
+
+ReactGA.initialize('UA-118996389-1');
+
+const history = createHistory();
+history.listen((location) => {
+  ReactGA.set({ page: location.hash });
```

Comment:
What is this needed for? Is `pageview` not enough?

Reply:
Single page applications should update the tracker with the set command in addition to sending a pageview when navigation pages dynamically. 

https://developers.google.com/analytics/devguides/collection/analyticsjs/single-page-applications